app/models/ncr/work_order.rb

```diff
@@ -120,11 +122,21 @@ def current_approver_email_address
         self.currently_awaiting_approvers.first.email_address
       elsif self.approving_official
         self.approving_official.email_address
+      elsif self.emergency and self.approvers.empty?
+        EMERGENCY_APPROVER_EMAIL
```

Comment:
This method could alternatively be called `email_address_of_the_person_the_proposal_is_waiting_on`, which for an `emergency` is no one. `nil` makes sense to me in that case. Totally open to other ideas, but returning an arbitrary string that can't (easily) be disambiguated from an email address seems more confusing.